app/models/order.rb

```diff
@@ -1,6 +1,8 @@
 require 'pago'
 class Order < ApplicationRecord
+   belongs_to :user, foreign_key: :users_id
    has_many :line_items, dependent: :destroy
+   scope :by_date , ->(from = Time.now.midnight,to = Time.now){where( created_at: (from..to))}
```

Comment:
what is the diff between Time.now and Time.current

Reply:
Time.now returns Time according to the local machine's time where as TIme.current returns time according to timezone set in Rails.